xml/ha_maintenance.xml

```diff
@@ -358,6 +370,29 @@ Node <replaceable>&node2;</replaceable>: standby
   </procedure>
 </sect1>
 
+<sect1 xml:id="sec-ha-maint-cluster-stop">
+ <title>Stopping the cluster services for the whole cluster</title>
+ <para>
+  To stop the cluster services on all nodes at once, use the following command:
+ </para>
+<screen>&prompt.root;<command>crm</command> cluster stop --all</screen>
+ <para>
+  To start the cluster services again after your maintenance work is done, use
+  the following command:
+ </para>
+<screen>&prompt.root;<command>crm</command> cluster start --all</screen>
+ <warning>
+  <title>Graceful shutdown not guaranteed</title>
+  <para>
+   The <option>--all</option> option alone does not guarantee graceful shutdown
+   of the cluster. Failure to stop the resources on a node might happen at the
+   application level, which is out of the scope of <option>--all</option>.
+   If a graceful shutdown is critical for your task, you might need to take additional
+   steps before stopping all of the cluster services. <!--Like what?-->
```

Comment:
What kind of steps might be involved here? Is this something I can document? Or is it different for every user?